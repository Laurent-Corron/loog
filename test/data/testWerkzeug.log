2019-12-19 08:42:32,135 19627 INFO ? odoo.addons.queue_job.jobrunner.runner: database connections ready
2019-12-19 08:42:32,892 19649 INFO odoo13-dev werkzeug: 127.0.0.1 - - [19/Dec/2019 08:42:32] "GET /web/static/lib/bootstrap/scss/web.assets_common/_functions.scss.css HTTP/1.1" 304 - 3 0.002 0.006
2019-12-19 08:42:35,044 19647 INFO odoo13-dev werkzeug: 127.0.0.1 - - [19/Dec/2019 08:42:35] "POST /web/dataset/call_kw/res.users/systray_get_activities HTTP/1.1" 200 -
2019-12-19 08:42:35,044 19647 INFO odoo13-dev werkzeug: 127.0.0.1 - - [19/Dec/2019 08:42:35] "POST /web/dataset/call_kw/res.users/systray_get_activities HTTP/1.1" 200 - 19 0.018 0.025
