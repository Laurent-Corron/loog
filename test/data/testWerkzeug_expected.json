{
	"asctime": "2019-12-19 08:42:32,892",
	"pid": "19649",
	"levelname": "INFO",
	"dbname": "odoo13-dev",
	"logger": "werkzeug",
	"message": "127.0.0.1 - - [19/Dec/2019 08:42:32] \"GET /web/static/lib/bootstrap/scss/web.assets_common/_functions.scss.css HTTP/1.1\" 304 - 3 0.002 0.006",
	"raw": "2019-12-19 08:42:32,892 19649 INFO odoo13-dev werkzeug: 127.0.0.1 - - [19/Dec/2019 08:42:32] \"GET /web/static/lib/bootstrap/scss/web.assets_common/_functions.scss.css HTTP/1.1\" 304 - 3 0.002 0.006\n",
	"werkzeug": {
		"remote_addr": "127.0.0.1",
		"request_method": "GET",
		"request_uri": "/web/static/lib/bootstrap/scss/web.assets_common/_functions.scss.css",
		"status": "304",
		"sql_count": "3",
		"sql_time": "0.002",
		"python_time": "0.006"
	}
}
